Makefile

```diff
@@ -68,6 +68,9 @@ check-requirements: .make.docker.pull
 compile-requirements: .make.docker.pull
 	${DC} run --rm compile-requirements
 
+install-local-python-deps:
```

Comment:
did you mean to include this unrelated change or is it in need of a rebase?

Reply:
Ah, apols @pmac - it was a deliberate addition (matching the convenience commands added to Basket and Bedrock), but I should have put it in its own commit to make it clear it was deliberate

Reply:
Cool. Just making sure 👍